leapyear.c

```diff
@@ -1,3 +1,6 @@
+/*to find the entered year is leap year or not*/
+
+
 #include <stdio.h>
 #include <stdlib.h>
 
```

Comment:
Why we need first two if condition where you are using 400 and 100 ?

Reply:
because to find if its a leap year or not it should be divisible by 400 and 4, and not divisible by 100. so we used if conditions to find if it is divisible by 400,100,4.